wdl/methods/phasing/Helper.wdl

```diff
@@ -392,11 +393,12 @@ task Shapeit4 {
         shapeit4.2 --input ~{vcf_input} \
                 --map ~{mappingfile} \
                 --region ~{region} \
-                --use-PS 0.0001 \
+                --use-PS ~{ps} \
                 --sequencing \
                 --output ~{prefix}.bcf \
                 --thread ~{num_threads} \
-                --log phased.log
+                --log phased.log \
+                --seed 123456
```

Comment:
Let's just use extra_args = `--use-PS 0.0001 --seed 123456` to handle cases like this.

Reply:
Also, just noticed the log---do we persist this anywhere?

Reply:
since we have already have the log file, this log file could be an extra. Would you suggest to delete this option?